File: languages/php/Bloock/Entity/Identity/MultichoiceAttribute.php
<?php

namespace Bloock\Entity\Identity;

class MultichoiceAttribute extends Attribute
{
    public static function fromProto(\Bloock\MultichoiceAttribute $res): MultichoiceAttribute
    {
        return new MultichoiceAttribute($res->getId(), $res->getValue());
    }

    public function toProto(): \Bloock\MultichoiceAttribute
    {
        $p = new \Bloock\MultiChoiceAttribute();
        $p->setId($this->id);
        $p->setValue($this->value);
        return $p;
    }
}

File: languages/python/bloock/entity/identity/multichoice_attribute.py
from __future__ import annotations

import bloock._bridge.proto.identity_entities_pb2 as proto
from bloock.entity.identity.attribute import Attribute


class MultichoiceAttribute(Attribute):
    @staticmethod
    def from_proto(a: proto.MultiChoiceAttribute) -> MultichoiceAttribute:
        return MultichoiceAttribute(
            id=a.id,
            value=a.value,
        )

    def to_proto(self) -> proto.MultiChoiceAttribute:
        return proto.MultiChoiceAttribute(
            id=self.id,
            value=self.value,
        )
